src/api/app/models/user_ldap_strategy.rb

```diff
@@ -591,4 +582,18 @@ def self.try_ldap_con(server, user_name, password)
     Rails.logger.debug("Bound as #{user_name}")
     conn
   end
+
+  private
+
+  def check_groups(relationships, method_name)
```

Comment:
Would be nice to have a more descriptive name, i.e. `relationship_groups_contains_user?`